tests/applications/Applications.ExtractImages.Tests/CohortCsvParserTests.cs

```diff
@@ -50,8 +50,7 @@ public void HappyPath(ExtractionKey expectedExtractionKey)
             var parser = new CohortCsvParser(fs);
             (ExtractionKey extractionKey, List<string> ids) = parser.Parse("foo.csv");
 
-            Assert.AreEqual(extractionKey, extractionKey);
```

Comment:
Should be check against `expectedExtractionKey`, I think.

Reply:
Good point, fixed.